images/linux/scripts/installers/azure-devops-cli.sh

```diff
@@ -4,12 +4,25 @@
 ##  Desc:  Installed Azure DevOps CLI (az devops)
 ################################################################################
```

Comment:
above comments can be applied for this script too I believe, not to double them :)